build.sbt

```diff
@@ -73,10 +79,16 @@ lazy val driver = module("driver", "driver/sources")
     libraryDependencies += Dependencies.nuProcess,
     libraryDependencies += Dependencies.remoteRobot,
     libraryDependencies += Dependencies.remoteRobotFixtures,
-    buildInfoKeys := Seq[BuildInfoKey](version, intellijBuild, "robotVersion" -> Dependencies.remoteRobot.revision),
+    buildInfoKeys := Seq[BuildInfoKey](
+      version,
+      BuildInfoKey.map(intellijBuild) { case (k, v) => k -> (v + "-EAP-SNAPSHOT") },
```

Comment:
It will fail if the version is like `2020.1`. Can we reuse the logic from the plugin itself here? They are appending the suffix only to the EAPs right?